bootstrap.sh

```diff
@@ -16,20 +16,48 @@
 
 ############################  SETUP PARAMETERS
 app_name='spf13-vim'
-[ -z "$APP_PATH" ] && APP_PATH="$HOME/.spf13-vim-3"
-[ -z "$REPO_URI" ] && REPO_URI='https://github.com/spf13/spf13-vim.git'
-[ -z "$REPO_BRANCH" ] && REPO_BRANCH='3.0'
-debug_mode='0'
-fork_maintainer='0'
-[ -z "$VUNDLE_URI" ] && VUNDLE_URI="https://github.com/gmarik/vundle.git"
 
+if [[ -z "$XDG_CONFIG_HOME" ]]; then
+    [[ -z "$SPF13_VIM" ]] && SPF13_VIM="$HOME/.spf13-vim-3"
```

Comment:
I changed $APP_PATH to $SPF13_VIM as I feel like we should move towards prefixing any environment variables that can interact with the app with the name of the app instead of some cryptic name...